ricecooker/config.py

```diff
@@ -14,13 +14,78 @@
 SUSHI_BAR_CLIENT = None
 STAGE = False
 
-# Setup logger
+# Don't use this - call logging.getLogger(__name__) from each
+# individual module. Logging is configured centrally by calling
+# setup_logging()
 LOGGER = logging.getLogger()
```

Comment:
Consider this slightly modified simple chef script [this script](https://gist.github.com/ivanistheone/df1ac717993d1c8e97d78cc52c1f0702)

<img width="721" alt="Screen Shot 2020-04-02 at 10 06 50 AM" src="https://user-images.githubusercontent.com/163966/78259207-5e263700-74ca-11ea-92a4-249df7b7433c.png">


### Expected
To see both the log lines, first: `This is a log message par of the script but before chef has started` and later `This is a log message that is part of the chef run`.

### Actual
Only the second log line appears because LOGGER is not setup before chef runs.

Reply:
This is fixed now, simply by setting up the default logging at module load time